src/app/mods/salesSummary/salesSummary.ts

```diff
@@ -0,0 +1,141 @@
+
+        // Create contentBox to display total kamas/quantity sold
+        this.resumeBox = WindowContentHelper.createContentBox(this.wGame, 'sls-smy-total');
+        const qtySold: HTMLDivElement = this.wGame.document.createElement('div');
+        qtySold.className = 'sls-smy-total-text';
+        const totalPrice: HTMLDivElement = this.wGame.document.createElement('div');
+        totalPrice.className = 'sls-smy-total-text';
+        this.resumeBox.append(qtySold, totalPrice);
+
+        // Add element to window
+        this.window.addContent(this.table.getHtmlElement).addContent(this.resumeBox);
+    }
+
+    private onTextInformationMessage() {
+        this.on(this.wGame.dofus.connectionManager, 'TextInformationMessage', async (e:any) => {
+            if (e.msgId == 73) {
+                this.awaitingSales.push({id: e.parameters[1], price: e.parameters[0], quantity: e.parameters[3]});
+
+                // Use debounce to be sure to get all data before load
+                if (this.debounce) clearInterval(this.debounce);
+                this.debounce = setTimeout(() => {
+                    this.loadData().then(() => {
+                        this.table.insertData(this.sales);
+                        this.resumeBox.children[0].insertAdjacentText('beforeend', `Item(s) vendu : ${this.sales.length}`);
+                        this.resumeBox.children[1].insertAdjacentText('beforeend', `Total gains : ${this.formatNumber(this.totalSales)} k`);
+                        this.window.show();
+                    });
+                }, 500);
+            }
+        });
+    }
+
+    /**
+     * Load data for all entry in awaitingSales, add await to load one time a same item
+     */
+    private async loadData() {
+        for await (const item of this.awaitingSales) {
+            if (this.resourcesKnow[item.id] == null) {
+                const res = await axios.post('https://proxyconnection.touch.dofus.com/data/map?lang=fr&v=1.49.9', {class: "Items", ids: [item.id]});
+                this.resourcesKnow[item.id] = {name: res.data[item.id].nameId, iconId: res.data[item.id].iconId};
+            }
+
+            this.createItem(this.resourcesKnow[item.id], item.price, item.quantity);
+        }
+    }
+
+    /**
+     * Create the HTMLDivElement for the item image and push data to final array
+     */
+    private createItem(item: Item, price: number, quantity: number) {
+        const img: HTMLDivElement = this.wGame.document.createElement('div');
+        img.className = 'sls-smy-img';
+        img.insertAdjacentHTML('afterbegin', `<img src="https://dofustouch.cdn.ankama.com/assets/2.34.8_kbu_6h45kmUJaqYJSzE(uwaos..pYYKs/gfx/items/${item.iconId}.png">`);
+    
+        this.totalSales += +price;
+        this.sales.push([img, item.name, quantity, this.formatNumber(price) + ' k']);
+    }
+
+    private formatNumber(number: number): string {
+        return number.toString().replace(/\B(?=(\d{3})+(?!\d))/g, " ");
+    }
+    
+}
+
```

Comment:
You have to implement the function `reset()`

    public reset() {
        this.window.destroy();
        this.wGame.document.getElementById('salesSummaryCss').remove();
        super.reset()
    } 

Thans to Xanadu